serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -57,55 +164,162 @@ Double-check your `$\{HOME\}/.docker/config.json`. If you're using local desktop
 
 === Configure the {operator_name} (i.e. registry address, secret) for building your Workflows
 
-The `KogitoServerlessPlatform` is the Custom Resource used to control the behavior of the {operator_name}.
+The `SonataFlowPlatform` is the Custom Resource used to control the behavior of the {operator_name}.
 It defines the behavior of the operator when handling all {product_name} Custom Resources (Workflow and Build) in the given namespace.
 
-Since the operator is installed in global mode, you will need to specify a `KogitoServerlessPlatform` in each Namespace where you want to deploy Workflows.
-You can find a basic `KogitoServerlessPlatform` Custom Resource example in the link:https://github.com/kiegroup/kogito-serverless-operator/tree/{operator_version}/config/samples[`config/samples` folder] that you can simply apply to configure your operator.
+Since the operator is installed in global mode, you will need to specify a `SonataFlowPlatform` in each Namespace where you want to deploy Workflows.
 
-.Create a `KogitoServerlessPlatform`
-[source,bash,subs="attributes+"]
+[TIP]
+====
+If you have deployed a workflow for xref:cloud/operator/developing-workflows.adoc[development] you already have a `SonataFlowPlatform` custom resource. Just edit the one you have.
```

Comment:
In that guide there is no step of creating a platform unless you want to specify a different devBaseImage.

Reply:
The operator creates one.